mkdud

```diff
@@ -430,11 +438,13 @@ Create driver update:
       --no-docs                 Don't include package documentation in unpacked instsys tree
                                 (to save space). This is the default setting.
       --keep-docs               Include package documentation in unpacked instsys tree.
-      --force                   Override some internal consistency checks. This allows you to
-                                build driver updates containing files that might break the
-                                installation. mkdud will normally remove those files and
-                                print a warning. This also allows you to specify distributions
-                                with --dist that mkdud does not know about.
+      --force                   Obsolete. Use one or more of --no-fix-XXX instead.
+      --no-fix-yast             Allow driver update to replace /sbin/yast. See 'Consistency checks'.
```

Comment:
I'm bit puzzled here, `--no-fix-yast` allows fixing `/sbin/yast`?

I'd expect `--no-fix-yast` with "Do not allow driver update to replace /sbin/yast" or `--fix-yast` with "Allow..." description.

Reply:
`fix-yast` would 'fix' the yast package. So `no-fix-yast` avoids fixing the yast package. See also the detailed paragraph below.

Reply:
OK, got it.